packages/react-relay/modern/__tests__/ReactRelayQueryRenderer-test.js

```diff
@@ -166,11 +166,9 @@ describe('ReactRelayQueryRenderer', () => {
           },
         },
       };
-      const fetch = () => {
-        return {
-          kind: 'data',
-          data: response,
-        };
+      const fetch = (operation, variables, cacheConfig, uploadables, observer) => {
+        observer.onCompleted(response);
+        return {};
```

Comment:
Realized after doing this that the `RelayNetwork.create()` should not pass the internal observer object through to the user-provided fetch function, will change this.